src/analyzer/protocol/bittorrent/BitTorrentTracker.cc

```diff
@@ -243,43 +243,67 @@ void BitTorrentTracker_Analyzer::DeliverWeird(const char* msg, bool orig)
 bool BitTorrentTracker_Analyzer::ParseRequest(char* line)
 	{
 	static bool initialized = false;
-	static regex_t r_get, r_get_end, r_hdr;
+	static RE_Matcher re_get("^GET[ \t]+");
+	static RE_Matcher re_get_end("[ \t]+HTTP/[0-9.]+$");
```

Comment:
This was already present in the existing code, but I think we should have this instead:
```suggestion
	static RE_Matcher re_get_end("[ \t]+HTTP/[0-9\.]+$");
```
Or am I missing something?